roofit/roofitcore/inc/RooAbsData.h

```diff
@@ -360,10 +375,12 @@ class RooAbsData : public TNamed, public RooPrintable {
 
   std::unique_ptr<RooArgSet> _globalObservables; // Snapshot of global observables
 
+  mutable TNamed* _namePtr ; //! De-duplicated name pointer. This will be equal for all objects with the same name.
```

Comment:
```suggestion
  mutable const TNamed * _namePtr ; //! De-duplicated name pointer. This will be equal for all objects with the same name.
```
I *think* you can make this `const`, as you never want to interfere with the name that's behind it.

Reply:
It's copy-paste from RooAbsArg. Should I try changing it in RooAbsArg as well ?

Reply:
If you are up for modernising RF, absolutely!